.env.example

```diff
@@ -2,6 +2,7 @@ BASE_URL=http://localhost:5000
 DATABASE_URL=postgres://<user>:<pass>@localhost:5432/goodtables
 TEST_DATABASE_URL=postgres://<user>:<pass>@localhost:5432/goodtables_test
 BROKER_URL=redis://localhost:6379/10
+RESULT_BACKEND=redis://localhost:6379/11
```

Comment:
Another reason to not use the current syncing approach :) Ok, we will probably get rid of this at some point but if we have to stick with it we might want to investigate if we can store only the syncing tasks results and not the validation ones.